### ISSUE TITLE
Problem with Resizable only injecting width on refresh

### ISSUE THREAD

**Issue by NashBaughn** [esnet/react-timeseries-charts#334]
Lovely package, I just can't seem to get Resizable to work properly. On window resize, the chart is unable to update its width. However, if the page is refreshed, Resizable is able to do its job just fine. 

It may be a little unnecessary, but I have added some screenshots in order to throughly describe my issue. 

The image below shows the page when it is first loaded, notice how Resizable properly inject the width of the charts parent.
<img width="500" alt="screen shot 2018-10-16 at 2 57 42 pm" src="https://user-images.githubusercontent.com/19157432/47050086-08ce5c80-d154-11e8-872c-15afe840ecd6.png">

This next image shows what happens after the screen is resized. The width of the svg remains the same as it appears Resizable is not functioning as it should. 
<img width="500" alt="screen shot 2018-10-16 at 2 58 02 pm" src="https://user-images.githubusercontent.com/19157432/47050136-1f74b380-d154-11e8-8b41-cb0f1e8b7326.png">

Finally, this image shows the result after the page is reloaded and Resizable is able to inject the correct width. (but only on reload!)
<img width="500" alt="screen shot 2018-10-16 at 2 58 17 pm" src="https://user-images.githubusercontent.com/19157432/47050165-387d6480-d154-11e8-90d5-38320a9d1adf.png">

It is worth noting that I am using 'flex' to achieve component resize for when the window size changes. I just cant for the life of me get this to work. Any feedback is appreciated it! 

Thanks

**Comment by pjm17971** [comment]
Ok, I've seen a few similar issues over the years. I guess as this is what interfaces with user code, it's hard to tell exactly what isn't triggering and it's also potentially browser sensitive.

Can you hack up the node_modules/react-timeseries-charts/lib/components/Resizable.js code with some debugging and see if you can figure out why the handleResize() call isn't firing when you resize the window?

The code is pretty simple (though obviously transpiled), it just uses a window event listener to set this.state.width, which is then injected into the top level component (ChartContainer) with the clone stuff.
See https://developer.mozilla.org/en-US/docs/Web/Events/resize

Also, what browser are you using?

**Comment by NashBaughn** [comment]
I will give that a shot. I have been running the website locally next to my project in order to somewhat replicate the DOM structure. My browser is Chrome 69. The test website works just as expected with dynamic resizing. 

Ill post an update in a couple hours after I have hopefully narrowed down the issue.

**Comment by NashBaughn** [comment]
So... I believe I have found something noteworthy. Like I said I had been comparing my own project and the example website to see where I was going wrong. 

I noticed that 'src/website/packages/charts/examples/barchart/Index.js' did not import from 'react-timeseries-charts', but rather from a local path. 

After changing the name is package.json so I could npm install 'react-timeseries-charts', I made the below change. (Line 26 and 27). 
![screen shot 2018-10-16 at 7 25 27 pm](https://user-images.githubusercontent.com/19157432/47058421-d5062d80-d179-11e8-8782-c3b9f3fe2196.png)

This created the same problem I have been having. Resizable works on page load/refresh, but not on resize. 

Pretty easy to re-create if needed.

Does this make sense?
